### PR TITLE
Sitemap generator

### PR DESCRIPTION
**Pull request by jamesjefferies**

### CODE REVIEW

File: config/sitemap.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    add datasets_path
+    add team_path
+    add getting_started_path
+    add scoring_path
+    add privacy_and_cookie_policy_path
+
+    add schools_path
+    add activity_types_path
+
+    ActivityType.find_each do |activity_type|
```
Comment (jamesalmond) [inline config/sitemap.rb]: Only active?

File: config/app/controllers/robots_txts_controller.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class RobotsTxtsController < ApplicationController
```
Comment (jamesalmond) [inline config/app/controllers/robots_txts_controller.rb]: Is this a duplicate file?

Comment (jamesjefferies) [reply config/app/controllers/robots_txts_controller.rb]: Yes!

File: config/application.rb [hunk 3]
```diff
@@ -3,7 +3,7 @@
 require "rails"
 require "rails/all"
 require "active_storage/engine"
-require_relative "../lib/rack_x_robots_tag"
+require_relative "../lib/rack/x_robots_tag"
```
Comment (jamesalmond) [inline config/application.rb]: @jamesjefferies Something to do with autoloading doesn't like that the file name wasn't related to the class name and meant test kept cycling and wouldn't load.

Comment (jamesjefferies) [reply config/application.rb]: woah, that's a bit weird isn't it? I didn't even look to make sure the test deploy had worked - more fool me... you fixed it then?

Comment (jamesjefferies) [reply config/application.rb]: thanks :)

Comment (jamesalmond) [reply config/application.rb]: Yes, this seems to fix it!

**Pull request merged**